olap/src/aggregator/ceramic_patch.rs

```diff
@@ -60,19 +61,32 @@ impl WindowUDFImpl for CeramicPatch {
         Ok(Box::new(CeramicPatchEvaluator))
     }
 }
+// Small wrapper container around the data/state fields to hold
```

Comment:
can we de-dupe this type with the one that is inline in the function above?

Reply:
No, this type is different in key ways. First its in an entirely separate crate (i.e. the client crate vs the server crate). Second the type here expects the data to be json patch data where the server handles it as opaque data. As this is the client specific code for aggregating specifically MIDs it makes sense for it to have its own specialized type to understand the data.

Reply:
In that case, should we call this type `ModelInstanceDocumentDataContainer` or something like that so it's clear that it's MID specific?

Reply:
Yes, I went for MIDDataContainer to keep the name shorter and add a comment explaining its specific to model instance documents.